pallet-tft-bridge/src/lib.rs

```diff
@@ -34,6 +29,9 @@ pub trait Config: system::Config {
 
 	/// Handler for the unbalanced decrement when slashing (burning collateral)
 	type Burn: OnUnbalanced<NegativeImbalanceOf<Self>>;
+
+	/// Origin for sudo extrinsics
+	type CouncilOrigin: EnsureOrigin<Self::Origin>;
```

Comment:
I would try to find a better name here, besides the fact that "Council" is an internal working term  to grasp the DAO better, from the pallet's perspective it is an Origin that protects the restricted functionality and is configured externally ( can be a combination of origin rules).